### DISCUSSION TITLE
Change size of eeprom to 256 bytes

### DISCUSSION THREAD

**Discussion by manhabc** [dbuezas/lgt8fx#283] [General]
Hi, is this possible to change size of eeprom from (let say 1024bytes) to 256 bytes?
My program used only 100 byte for eeprom. And I need more space for main sketch?

**Comment by LaZsolt** [comment]
A size smaller than 1024 is not possible because of erase size.

**Reply by RudolfAtHome** [reply]
Instead, is it possible to completely dispense with the EEPROM in the flash? There are many applications (including mine) that do not require EEPROM. 1 kByte more Flash can often help!
